src/amplitude/events.ts

```diff
@@ -45,3 +45,11 @@ export function sendCheckboxFjernet(label?: string) {
 export function sendPanelEkspanderEvent(panelnavn: string) {
   logEvent("panel-ekspander", { panelnavn });
 }
+
+export function sendOppgaveStatusEvent(status: string, oppgavetittel: string) {
+  logEvent("knapp", { status, oppgavetittel });
+}
+
+export function sendÅpneAktivitetEvent(oppgavetittel: string) {
+  logEvent("åpne", { oppgavetittel });
```

Comment:
Hvis vi skal følge "taksonomien" som er satt opp, så er det "[accordion åpnet](https://github.com/navikt/analytics-taxonomy/blob/main/events/accordion%20%C3%A5pnet/README.md))" og event-property "tekst" som er riktig navngivning for denne eventen

Reply:
Bra spotta! Det er pushet endring.